### PR TITLE
Adding sample code

### PR DESCRIPTION
**Pull request by rishikav**
Adding sample bhai-lang code for fibonacci series.

### CODE REVIEW

**Review by abhi1510abhi — APPROVED** [review]

**Review by impratik7 — APPROVED** [review]
Sounds too good 😄 👍

**Review by aniketsingh0104 — CHANGES_REQUESTED** [review]
Nit : Fix indentation.

File: sample/Fibonacci.bhai [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+bhai ye hai n = 0;
+jab tak bhai (n < 10) {
+  bhai ye hai nextKaNext = start + next;
+  bol bhai nextKaNext;
+  start = next;
+  next = nextKaNext;
+  n += 1;
+  }
+
+bye bhai
```
Comment (ritikbanger) [inline sample/Fibonacci.bhai:15]: Update your file with this code. Though BhaiLang can work without indentation, but as suggested, you can use it.

```
hi bhai

  bol bhai "Fibonacci likhenge";
  bhai ye hai start = 0;
  bhai ye hai next = 1;
  bol bhai start;
  bol bhai next;
  bhai ye hai n = 0;
  jab tak bhai (n < 10) {
    bhai ye hai nextKaNext = start + next;
    bol bhai nextKaNext;
    start = next;
    next = nextKaNext;
    n += 1;
  }

bye bhai
```

**Comment by rishikav** [comment]
I have made the requested changes but I am unable to understand the error. Could you please help out?

**Comment by aniketsingh0104** [comment]
Hey. Thanks for this but as of now we won't be adding this in this repo. If you wish you can create a separate repo for these sample codes like this - https://github.com/carrotfarmer/bhai-lang .

**Pull request closed without merging**